autonomy/chain/tx.py

```diff
@@ -66,6 +66,8 @@
     "already_exists",
     "already known",
     "oldnonce",
+    "replacement transaction underpriced",
+    "is contract deployed correctly and chain synced?",
```

Comment:
Are we sure about this one here ? This can happen for several reasons:
1. Wrong contract address (real error, tx settler should fail)
2. Out of sinc RPC waiting a recent contract deployment (RPC issue, should retry)

Reply:
Oh you mean if the contract address is wrong, yeah in that case the error will be "is contract deployed". I don't know yet how we can differentiate between the two, but I'd rather have it retry before eventually failing instead of not retrying at all. Because this way, it will be able to handle unsynced RPC with the cost of delay in the failure that is actually because of a "programming" error. Since, the wrong contract error is not flaky, we can avoid that as developers.